tests/integration_tests_control_files/euler_entropy_conservation/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(TEST_OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
+
+# =======================================
+# Entropy conservation test 
+# =======================================
+# ----------------------------------------
+# - Runs a short time on TGV
+# - test will fail if entropy is not conserved
+# ----------------------------------------
+configure_file(euler_ismail_roe_entropy_check.prm euler_ismail_roe_entropy_check.prm COPYONLY)
+add_test(
+    NAME MPI_EULER_ISMAIL_ROE_ENTROPY_CHECK
+  COMMAND mpirun -np 1 ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_3D -i ${CMAKE_CURRENT_BINARY_DIR}/euler_ismail_roe_entropy_check.prm
```

Comment:
rename to "SERIAL", only using 1 processor

Reply:
This should be MPI - I pushed a version with only 1 processor by accident (see other comment regarding the MPI sum, I was testing whether that was correct). Fixing. I'll also add a `3D` specifier for better consistency with other tests.